### ISSUE TITLE
It don't work on Rails 3.x

### ISSUE THREAD

**Issue by phlegx** [aaronchi/jrails#15]
Hi!

This nice plugin don't work on Rails 3. Does anyone have any idea how to make it run?

Best regards

**Comment by gravis** [comment]
It doesn't make any sense to run jrails on rails3. Rails3 is agnostic now regarding JS frameworks, you must update your code accordingly.
regards

**Comment by phlegx** [comment]
OK, so I can remove jrails from the rails3 projects and the code have to run!?

NICE!!! :)))

**Comment by gravis** [comment]
you can remove the plugin, but you will certainly have to adapt your code

**Comment by phlegx** [comment]
What do you mean? For example?

I use e.g. remote_function...

**Comment by gravis** [comment]
you should really take a look at https://github.com/rails/jquery-ujs ;)

**Comment by phlegx** [comment]
Yes ok, but this don't replace a remote_function method that starts a ajax request if e.g. a select box changes. Or not?

**Comment by MikeEmery** [comment]
@gravis

While RJS seems to be falling out of favor in Rails 3, it still works in Rails 3 and by default Rails will use Prototype methods in RJS.

**Comment by korny** [comment]
Does it work in Rails 3.1? Anybody tried this before?
